React/Modules/RCTAsyncLocalStorage.m

```diff
@@ -223,16 +214,17 @@ - (NSDictionary *)_ensureSetup
 #endif
 
   NSError *error = nil;
-  if (!RCTHasCreatedStorageDirectory) {
+  // NOTE(nikki93): `withIntermediateDirectories:YES` makes this ok to do if directory already exists...
+//  if (!RCTHasCreatedStorageDirectory) {
     [[NSFileManager defaultManager] createDirectoryAtPath:RCTGetStorageDirectory()
                               withIntermediateDirectories:YES
                                                attributes:nil
                                                     error:&error];
     if (error) {
       return RCTMakeError(@"Failed to create storage directory.", error, nil);
     }
-    RCTHasCreatedStorageDirectory = YES;
-  }
+//    RCTHasCreatedStorageDirectory = YES;
```

Comment:
I'd just go ahead and remove rather than commenting these (and similar elsewhere in the diff)